rskj-core/src/main/java/co/rsk/net/sync/PeersInformation.java

```diff
@@ -178,19 +174,6 @@ private int comparePeerFailInstant(
         return otherFailInstant.compareTo(failInstant);
     }
 
-    private int comparePeerScoring(
```

Comment:
As an exception, I would probably leave these uncommented methods and not delete them. There were plans to use them at some point, for example [here](https://github.com/rsksmart/rskj/blob/06713cae2281a2070b202cd44cc2fd62d627c910/rskj-core/src/main/java/co/rsk/net/sync/PeersInformation.java#L66), so maybe we should keep them. WDYT?